RealmSwift-swift2.0/Realm.swift

```diff
@@ -279,8 +279,7 @@ public final class Realm {
     - returns: The created object.
     */
     public func create<T: Object>(type: T.Type, value: AnyObject = [:], update: Bool = false) -> T {
-        // FIXME: use T.className()
-        let className = (T.self as Object.Type).className()
+        let className = (type as Object.Type).className()
```

Comment:
yeah, the cast is still necessary. I don't see why we need the FIXME any more, as this fixes the issue with using the cast